NM trade enhancement.user.js

```diff
@@ -1594,13 +1594,26 @@ async function addTradePreview (notification) {
     singleton.setInstances(Object.values(tips));
 }
 
+/**
+ * Allows you to hit enter with your keyboard to dismiss confirm boxes.
+ * @param {Event} e - keydown event
+ */
+async function okayNotification (e) {
```

Comment:
The variable `e` is too short: in long code, it's no obvious for what it stands: event, exception, or error.